Library/Homebrew/extend/ENV/super.rb

```diff
@@ -126,20 +126,20 @@ def determine_path
     path.existing
   end
 
-  def homebrew_extra_pkg_config_paths
-    []
-  end
-
   def determine_pkg_config_path
     PATH.new(
       deps.map { |d| d.opt_lib/"pkgconfig" },
       deps.map { |d| d.opt_share/"pkgconfig" },
     ).existing
   end
 
+  def homebrew_extra_pkg_config_libdirs
```

Comment:
This doesn't seem to be an improvement to me. Can you explain more your motivation for this change?

Reply:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Sure, @MikeMcQuaid:  '`homebrew_extra_pkg_config_paths`' was used not inside its '`determine_pkg_config_path`' counterpart, but inside '`determine_pkg_config_libdir`' instead, so I moved it closer to its actual use within the same file and renamed it and its uses to match what it was actually doing for the sake of a minor code clarification.  (Why I was in this part of the codebase in the _first_ place is another story, but I digress…)